### PR TITLE
mv to aws-terraform cluster_profile

### PR DESCRIPTION
**Pull request by AsherShoshan**

### CODE REVIEW

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-presubmit

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-presubmit

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-presubmit

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-presubmit

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-presubmit

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 1]
```diff
@@ -93,10 +108,12 @@ tests:
...
       TF_FOLDER: ci/e2e/oidc_configuration/cluster_with_managed_oidc_config
       TF_VARS: |
-        url = "https://api.stage.openshift.com"
-        channel_group = "stable"
         provider_source  = "terraform-redhat/rhcs"
         provider_version =  ">= 1.1.0"
+        url = "https://api.openshift.com"
+        channel_group = "stable"
+        aws_region = "ap-northeast-1"
+        aws_availability_zones = ["ap-northeast-1a"]
```
Comment (amalykhi) [inline ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:116]: add
ocm_environment = "production"

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic2

**Comment by AsherShoshan** [comment]
/pj-rehearse ci/rehearse/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic

**Pull request closed without merging**